src/useFormState.test.tsx

```diff
@@ -433,6 +433,63 @@ describe("useFormState", () => {
     expect(autoSaves).toEqual(1);
   });
 
+  it("can queue up changes for auto save if a save is already in progress", async () => {
+    const autoSaveStub = jest.fn();
+    type FormValue = Pick<AuthorInput, "id" | "firstName" | "lastName">;
+    const config: ObjectConfig<FormValue> = {
+      id: { type: "value" },
+      firstName: { type: "value" },
+      lastName: { type: "value" },
+    };
+
+    // Given a component using `getObjectState` for lazily creating ObjectStates
+    function TestComponent() {
+      const [apiData, setApiData] = useState<FormValue>({ id: "a:1", firstName: "Brandon", lastName: "Dow" });
+      const state = useFormState({ config, autoSave, init: { input: apiData, map: (v) => v } });
+      async function autoSave(form: ObjectState<FormValue>) {
+        autoSaveStub(form.changedValue);
+        // Pretend to make an API call and update the local state
+        await setApiData((prevState) => ({ ...prevState, ...form.changedValue }));
+      }
+
+      return (
+        <div>
+          <button
+            data-testid="focusSetAndSaveField"
+            onClick={() => {
+              state.firstName.focus();
+              state.firstName.set("Foo");
+              state.firstName.maybeAutoSave();
+              state.firstName.blur();
+            }}
+          />
+          <button
+            data-testid="focusSetAndSaveFieldLastName"
+            onClick={() => {
+              state.lastName.focus();
+              state.lastName.set("Bar");
+              state.lastName.maybeAutoSave();
+            }}
+          />
+        </div>
+      );
+    }
+
+    const r = await render(<TestComponent />);
+    // And triggering the auto save behavior before awaiting the initial promise to resolve so we have pending changes.
+    click(r.focusSetAndSaveField());
+    click(r.focusSetAndSaveFieldLastName());
+    // Awaits the promises for all methods triggered above
+    await wait();
+    // Awaits the two `setTimeout`s triggered by the `pendingAutoSave` flag. (one for the `state.firstName.blur` and another for `state.lastName.maybeAutoSave`
```

Comment:
Nit I believe we don't have `setTimeout`s anymore, so update the comment? And maybe remove one of the `await wait`s...